tests/test_integration.py

```diff
@@ -16,7 +16,7 @@ def test_start_stop_status():
 
         cl = docker.client.from_env()
         containers = cl.containers.list()
-        assert len(containers) == 6
+        assert len(containers) == 7
```

Comment:
When managing merging this and #7, this will need updating, and in a couple of tests above